src/web/components/ReaderRevenueLinks.tsx

```diff
@@ -114,41 +120,77 @@ const subMessageStyles = css`
     margin: 5px 0;
 `;
 
+const month = new Date().getMonth() + 1;    // js date month begins at 0
+
 export const ReaderRevenueLinks: React.FC<Props> = ({
     edition,
     urls,
     dataLinkNamePrefix,
     inHeader,
+    pageViewId,
 }) => {
-    const showAusMomentHeader = edition === 'AU';
+    const ABTestAPI = useAB();
+
+    const getTestVariant = (): GlobalEoyHeaderTestVariant => {
+        if (inHeader && edition !== 'US') {
+            if (ABTestAPI.isUserInVariant(
+                GlobalEoyHeaderTestName,
+                'control',
+            )) {
+                return 'control';
+            }
+            if (ABTestAPI.isUserInVariant(
+                GlobalEoyHeaderTestName,
+                'variant',
+            )) {
+                return 'variant';
+            }
+        }
+        return 'notintest'
+    };
 
-    if (shouldHideSupportMessaging()) {
-        if (showAusMomentHeader) {
-            return (
-                <div className={cx(inHeader && paddingStyles)}>
-                    <div
-                        className={cx({
-                            [hiddenUntilTablet]: inHeader,
-                        })}
-                    >
-                        <div className={messageStyles}>Thank you</div>
-
-                        <div className={subMessageStyles}>
-                            <ThemeProvider theme={buttonBrand}>
-                                <LinkButton
-                                    priority="secondary"
-                                    showIcon={true}
-                                    size="small"
-                                    href="https://support.theguardian.com/aus-2020-map?INTCMP=Aus_moment_2020_frontend_header"
-                                >
-                                    Hear from other supporters
-                                </LinkButton>
-                            </ThemeProvider>
-                        </div>
-                    </div>
-                </div>
-            );
+    const variantName: GlobalEoyHeaderTestVariant = getTestVariant();
+
+    useEffect(() => {
+        if (variantName !== 'notintest') {
+            sendOphanComponentEvent('VIEW', {
+                abTestName: GlobalEoyHeaderTestName,
+                abTestVariant: variantName,
+                campaignCode: 'header_support',
+                componentType: 'ACQUISITIONS_HEADER',
+            });
         }
+    }, [variantName]);
+
+    const getHeading = (): string | JSX.Element => {
+        if (variantName === 'variant') return month === 12 ? `Support us this December` : 'Support us for 2021';
+        return <span>Support The&nbsp;Guardian</span>;
+    };
```

Comment:
is there a reason why we have a `span` for some text, and not for others?

Reply:
this line needs to keep the `&nbsp;` from the original, whereas just a string is fine for the variant copy